OSS-New-Deployment-FSP-Setup.postman_collection.json

```diff
@@ -416,6 +416,77 @@
 			],
 			"protocolProfileBehavior": {}
 		},
+		{
+			"name": "SettlementModels",
```

Comment:
I think we should call this folder "Add Settlement Models" for consistency

Reply:
done, adjusted naming.

Reply:
adjusted in a follow-up PR: https://github.com/mojaloop/postman/pull/100